src/awfy.js

```diff
@@ -88,7 +88,7 @@ export const BENCHMARKS = {
         suite: 'kraken',
         platformSuffix: '-qr',
         option: 'opt',
-        extraOptions: ['e10s', 'fission', 'stylo'],
+        extraOptions: ['e10s', 'fission', 'stylo', 'webrender'],
```

Comment:
Why do we need to also set `webrender` for `fission`-enabled Firefox?

Reply:
WebRender is a prerequisite for Fission